source/configuration_manual/authentication/master_users.rst

```diff
@@ -34,9 +34,7 @@ whole new user (with eg. :ref:`authentication-static_user_database` ). ``result_
 doesn't work with PAM or LDAP without ``auth_bind=yes``, because both of them
 require knowing the user's password.
 
-``result_success=continue`` is especially useful with a :ref:`authentication-checkpassword` passdb because the
-script gets both the login and the master username as environment variables.
-Other passdbs see only the login username in ``%u``. In the future there will
+Passdbs see only the login username in ``%u``. In the future there will
 probably be another setting to make the user verification to be done from
 userdb.
```

Comment:
Just remove the whole paragraph. I'm not sure if the leftover text is somehow related to the removed text, but after removing the text the rest makes even less sense.